concepticondata/concepticon.tsv

```diff
@@ -3958,4 +3958,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3972	EXCITED	Emotions and values	Arousal or a feeling of enthusiasm about something.	Property	
 3973	DEPRESSED	Emotions and values	A mental state of despondency, unhappinness, or hopelessness.	Property	
 3974	UNHAPPY	Emotions and values	Feeling not cheerful or glad, sometimes due to a misfortune.	Property	
-3975	NERVOUS	Emotions and values	A mental state of nervousness or agitation. Also, being easily excited or irritated.	Property	
\ No newline at end of file
+3975	NERVOUS	Emotions and values	A mental state of nervousness or agitation. Also, being easily excited or irritated.	Property	
```

Comment:
It could be that the file is marked as changed because an empty line was added at the end. @martino-vic Could you check the difference between the `concepticon.tsv` on the master branch and on your fork? Maybe with `git diff`.

Reply:
I have no idea what leads to this strange behaviour, I mentioned it in https://github.com/concepticon/concepticon-data/pull/1236#issuecomment-1268597875 already. Happens now constantly somehow.